test-data/unit/check-super.test

```diff
@@ -280,6 +280,20 @@ class B(A):
 class C:
     a = super().whatever  # E: super() outside of a method is not supported
 
+[case testSuperWithObjectClassAsFirstArgument]
+class A:
+    def f(self) -> None:
+        super(object, self).f()  # E: super() outside of a method is not supported
```

Comment:
Hmm the error message doesn't sound right. Wouldn't a better message be something like "Target class has no base class".